webextension/js/background/messageHandler.js

```diff
@@ -112,6 +112,14 @@ const messageHandler = {
       // lets remember the last tab created so we can close it if it looks like a redirect
       this.lastCreatedTab = tab;
       if (tab.cookieStoreId) {
+        // Don't count firefox-default, firefox-private, nor our own confirm page loads
+        if (tab.cookieStoreId !== "firefox-default" &&
+            tab.cookieStoreId !== "firefox-private" &&
+            tab.url.indexOf("confirm-page.html") === -1) {
```

Comment:
Can we check instead for moz-extension protocol instead perhaps?

Reply:
👍